aries_cloudagent/messaging/credential_definitions/tests/test_routes.py

```diff
@@ -259,9 +260,14 @@ async def test_send_credential_definition_no_auto_endorse(self):
             }
         )
 
-        self.request.query = {"auto_endorse": "false"}
+        self.request.query = {
+            "create_transaction_for_endorser": "true",
+            "conn_id": "dummy",
```

Comment:
Since we attach an Endorser Name to the connection, can we ask for the Endorser Name instead of the Connection ID?